job/hardware.go

```diff
@@ -27,28 +27,28 @@ type ComponentsResponse struct {
 func (j Job) AddHardware(w http.ResponseWriter, req *http.Request) {
 	b, err := readClose(req.Body)
 	if err != nil {
-		joblog.Error(errors.Wrap(err, "reading hardware component body"))
+		j.Error(errors.Wrap(err, "reading hardware component body"))
```

Comment:
what do you think about using `j.Logger.Error` here? really helps make it clear that this is a log line.

Reply:
:thinking: yeah that makes more sense. `j.Error` doesn't actually log!

Reply:
fixed in 0b0614cabc013d3990d09d0a78022922a1685020